### DISCUSSION TITLE
Welcome to cgeo Discussions!

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#9516] [General]
Lets just make a test drive whether this new GIthub feature turns out to be useful for us.
Maybe it helps to avoid starting (offtopic) discussion in particular issues. We can discuss implementation topics here and cut off new specific issues from that (if I understand the working mode of this feature correctly).

    ✏️ Optional: Customize the content below to let your community know what you intend to use Discussions for.

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by eddiemuc** [comment]
Might be a learning process for us when to discuss in issues and when in this Board :-)

First thing I noticed: unfortunately Discussions seems not (yet) to be supported by the GitHub App?

**Reply by Narrat** [reply]
Probably something like the discussion regarding the Quick settings for the map view.
Every design study would be a post on its own and with the possibility to directly reply to any post it kinda is organized.
Or topics were no immediate solution is available. More open threads.

**Comment by Lineflyer** [comment]
> First thing I noticed: unfortunately Discussions seems not (yet) to be supported by the GitHub App?

No idea, the feature is still in beta phase.

**Comment by baiti** [comment]
I think this discussions feature is very useful. Frequently I did not feel comfortable to open up an "issue" for something that really is not an issue.

**Comment by Le-Juan** [comment]
Hi!

I think discussions would be a good thing, maybe even something like a discord server might work. ;)
However I wanted to start a QA thread, but I saw that I wasn't able to so, I'll introduce myself here.

I'm Jan-Willem, you can call me Juan, I'm a bigtime user of C:Geo and it inspired me to start a personal project. My personal project is quite similar to GeoKrety and I was hoping to find some answers here about how to connect to geocaching, or at least be able to couple the geocachecode to coordinates.

**Comment by devemux86** [comment]
_Can't users create new discussions like in other repositories?_

I published [Mapsforge 0.16.0](https://search.maven.org/search?q=g:org.mapsforge) in Maven.

Thanks for all the code contributions!

**Reply by moving-bits** [reply]
Thanks @devemux86 for notifying us, we will update our dependency.

**Reply by eddiemuc** [reply]
And thanks @devemux86 for the great support when working in mapsforge!
